test/e2e/antreapolicy_test.go

```diff
@@ -736,6 +750,10 @@ func TestAntreaPolicy(t *testing.T) {
 	defer teardownTest(t, data)
 	initialize(t, data)
 
+	t.Run("TestGroupValidateACNP", func(t *testing.T) {
```

Comment:
Why don't we make this more generic for Antrea Policy as a whole? There would be other spec validation e2e testcases to be added for both ACNP and ANP.

Reply:
will update